tools/clang/test/CMakeLists.txt

```diff
@@ -69,7 +69,9 @@ endif()
 
 # HLSL Change Begin
 # Explicitly overriding check-clang dependencies for HLSL
-set(CLANG_TEST_DEPS clang opt FileCheck count not)
+set(CLANG_TEST_DEPS dxc dxcompiler clang-tblgen llvm-config opt FileCheck count not ClangUnitTests)
+add_custom_target(clang-test-depends DEPENDS ${CLANG_TEST_DEPS})
+set_target_properties(clang-test-depends PROPERTIES FILDER "Clang tests")
```

Comment:
Typo - FOLDER

Reply:
Someday I'll learn to spell and type well... That day is clearly not today.